content/docs/user-guide/editor/entity-inspector.md

```diff
@@ -1,11 +1,28 @@
 ---
-description: ' Use the Entity Inspector to add components to entities and modify
-  their properties in Open 3D Engine. '
+linkTitle: Entity Inspector
 title: Entity Inspector
+description: Use Entity Inspector to add components to entities and modify their properties in Open 3D Engine (O3DE).
+weight: 300
 ---
 
 The **Entity Inspector** manages all the components for each entity. Select an entity in the **Entity Outliner** or the viewport to see the attached components in the **Entity Inspector**.
 
+Use the **Entity Inspector** to do the following:
+
+* Add components to entities
+* Modify component properties
+* Remove, copy, cut, and paste components
+* Set the entity status
+* Customize the entity icon
+* Pin an entity's inspector
+* Create your own help topic for your custom component
```

Comment:
update: Should these use '-' not '*' for consistency (I see we're standardizing on that and not '+' or '*')

Reply:
Hugo will render all asterisk, plus-sign, minus-sign, (and probably dash as well).  `-` is definitely preferred, and we switch over to it when we can.